tests/Install-Scripts.Test/GivenThatIWantToInstallDotnetFromAScript.cs

```diff
@@ -33,9 +33,11 @@ public class GivenThatIWantToInstallDotnetFromAScript : IDisposable
                 ("6.0", "6\\.0\\..*", Quality.Daily),
                 ("6.0", "6\\.0\\..*", Quality.None),
                 ("STS", "7\\.0\\..*", Quality.None),
-                ("LTS", "6\\.0\\..*", Quality.None),
```

Comment:
This test is relevant,  please leave it as it is and add a separate for 8.0.xx

Reply:
Thank you for the review! 
May I clarify why the test is relevant since the latest Long Term Support version is 8? 
Channel used for the test is LTS, according to the docs: https://learn.microsoft.com/en-gb/dotnet/core/tools/dotnet-install-script#options we are downloading the most recent version of long term support. 
Am I missing something?

Reply:
You can check the support of the versions on this page https://dotnet.microsoft.com/en-us/download/dotnet
As you can see, net 6.0 saves it's status "LTS". It happens in parallel to net 8.0 "LTS" marker :)

Reply:
Thank you for checking it! 
The status of net 6.0 is LTS and will be supported however it is not recent one, aka.ms links are updated to download the dotnet 8 version, since it is the recent one, sorry for the confusions if I'm missing something. 
Also without updating the LTS regular expression to 8 the tests autoamtically fail since it downloads the dotnet 8.

Reply:
Oh, I see, you are right, sorry for misleading you.

Reply:
Thank you for raising the question! resolving the conversation